### PR TITLE
Update AnnotationScanner.php

### LINKED ISSUE #5595: ANNOTATION_IGNORE for correct annotation string when using tab idents
Zend\Code\Scanner\AnnotationScanner doesn't support tabs.
If someone use tab indents instead of spaces (many projects in my company and we all use auto formatting in Netbeans) annotations will be marked as ANNOTATION_IGNORE.

Doctrine annotations support both ways of formatting.

### PR DESCRIPTION
**Pull request by nouc23**
I know we should write code using PSR standard but life is different.
While we use tabs script ignore annotations.

Fixes #5595

### CODE REVIEW

**Comment by Ocramius** [comment]
This requires a test asset containing tabs, and a related test

**Comment by nouc23** [comment]
done

**Comment by weierophinney** [comment]
Needs:

- Rebase
- new test.

File: tests/ZendTest/Code/Scanner/AnnotationScannerTest.php [hunk 1]
```diff
@@ -17,23 +17,20 @@
...
+     *
+     * @param string $docComment
      */
-    public function testScannerWorks($newLine)
+    public function testScannerWorks($docComment)
     {
         $annotationManager = new AnnotationManager();
         $parser = new GenericAnnotationParser();
-        $parser->registerAnnotations(array(
+        $parser->registerAnnotations([
```
Comment (Maks3w) [inline tests/ZendTest/Code/Scanner/AnnotationScannerTest.php]: Use PHP 5.3 array style

**Comment by Maks3w** [comment]
Fixes #5595

**Comment by Maks3w** [comment]
I've pulled this and found the test is not related with the change made in scanner. So I don't know what is trying to fix this. Test never fails.

**Comment by nouc23** [comment]
I edit existing test, and I was told to create new one.
I forgot to do that and also I doesnt use github much. I try to finish this up on days.
--
Wysłano z aplikacji myMail dla Androida
czwartek, 07 maja 2015, 03:57PM +02:00 od Maks3w <notifications@github.com>:

**Comment by GeeH** [comment]
This issue has been moved from the `zendframework` repository as part of the bug migration program as outlined here - http://framework.zend.com/blog/2016-04-11-issue-closures.html
New issue can be found at: https://github.com/zendframework/zend-code/issues/78

**Pull request closed without merging**